foodie-application/src/main/java/com/foodie/repository/HotelRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.foodie.repository;
+
+import com.foodie.entity.register.HotelRegistrationRequest;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface HotelRepository extends JpaRepository<HotelRegistrationRequest, String> {
```

Comment:
How come it will be string here, we need to give the  data type of primary key right, I see in entity its Long

Reply:
Yes, changed